CONTRIBUTING.md

```diff
@@ -2,43 +2,7 @@
 
 Anyone is welcome to contribute!
 
-## Getting Started
```

Comment:
I'm still working on this file, but I've moved this section to the Readme since getting started with the repo isn't just for contributors.

Reply:
Awesome! My thoughts too